helix-query.yaml

```diff
@@ -6,18 +6,12 @@ indices:
       - /**
     target: /query-index.json
     properties:
-      lastModified:
```

Comment:
I'd recommend not changing the query index file just yet. Instead, I'd probably approach this in phases. basically get all the code changes done, working and deployed to no longer use those fields. Once sure that is good, then change the index definition, the index spreadsheet, and finally run a re-index to bring the index up to date.

Reply:
helix-query.yaml reverted